vendor/src/github.com/docker/engine-api/client/image_build.go

```diff
@@ -110,6 +110,13 @@ func imageBuildOptionsToQuery(options types.ImageBuildOptions) (url.Values, erro
 		return query, err
 	}
 	query.Set("labels", string(labelsJSON))
+
+	cacheFromJSON, err := json.Marshal(options.CacheFrom)
+	if err != nil {
+		return query, err
+	}
+	query.Set("cachefrom", string(cacheFromJSON))
```

Comment:
What's the process around changing things in `engine-api`? I guess this needs a PR to the separate repo housing this, and a bump to the version of the client-daemon API.